SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDKFunctionalTests.Build.cs

```diff
@@ -21,5 +21,12 @@ public SpatialGDKFunctionalTests(ReadOnlyTargetRules Target) : base(Target)
                 "HTTP",
                 "UnrealEd"
             });
+
+        if (Target.bBuildEditor == true)
+        {
+            // For spatial functional tests
+            PrivateDependencyModuleNames.Add("UnrealEd");
+            PrivateDependencyModuleNames.Add("EngineSettings");
```

Comment:
I already added UnrealEd above
EngineSettings can probably go directly there as well, since this is currently an editor-only module (so Target.bBuildEditor is always true).

Reply:
Thanks, yes I think I missed that as it probably got auto merged. Good point about this being an editor-only module. I will move EngineSettings above too.